### PR TITLE
chore(package): update dependencies

### PR DESCRIPTION
**Pull request by nolanlawson**
## Details

Regularly-scheduled dependencies update.

## Does this PR introduce breaking changes?

* ✅ `No, it does not introduce breaking changes.`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,36 +26,36 @@
...
-    "eslint": "^7.29.0",
+    "eslint": "^7.30.0",
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jest": "^24.3.6",
     "eslint-plugin-lwc-internal": "link:./scripts/eslint-plugin",
     "execa": "^5.1.1",
     "fs-extra": "^10.0.0",
     "glob": "^7.1.7",
-    "husky": "^6.0.0",
+    "husky": "^7.0.1",
```
Comment (jodarove) [inline package.json:45]: From [Husky 7.x](https://github.com/typicode/husky/releases/tag/v7.0.0): 

> Improve .husky/ directory structure. .husky/.gitignore is now unnecessary and can be removed.

should we remove [ours](https://github.com/salesforce/lwc/blob/master/.husky/.gitignore)?

File: yarn.lock [hunk 2]
```diff
@@ -4,26 +4,26 @@
 
 "@babel/code-frame@7.12.11":
   version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
+  resolved "https://nexus.soma.salesforce.com/nexus/content/groups/npm-all/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
```
Comment (jodarove) [inline yarn.lock:7]: this doesn't seem correct.

**Review by jodarove — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nolanlawson) [reply package.json:45]: Good catch! I didn't notice; I was just looking for breaking changes. 😅

File: yarn.lock (hunk 2 above)
Comment (nolanlawson) [reply yarn.lock:7]: My mistake. I always forget to switch my `npmrc` back. 🤦

**Review by ravijayaramappa — APPROVED** [review]

**Review by jodarove — APPROVED** [review]
🚀

**Comment by nolanlawson** [comment]
Updated the deps-of-deps too, `yarn audit` is clean now ✨

**Review by jodarove — APPROVED** [review]
lgtm!

**Pull request merged**